modules/RTC/TraceablePeerConnection.js

```diff
@@ -693,11 +692,19 @@ TraceablePeerConnection.prototype.getAudioLevels = function(speakerList = []) {
 
 /**
  * Checks if the browser is currently doing true simulcast where in three different media streams are being sent to the
- * bridge. Currently this happens only when VP8 is the selected codec.
+ * bridge. Currently this happens always for VP8 and only if simulcast is enabled for VP9/AV1/H264.
  * @returns {boolean}
  */
 TraceablePeerConnection.prototype.doesTrueSimulcast = function() {
-    return this.isSimulcastOn() && this.getConfiguredVideoCodec() === CodecMimeType.VP8;
+    const currentCodec = this.getConfiguredVideoCodec();
+    const codecScalabilityModeSettings = this.tpcUtils.codecSettings[currentCodec];
+    const runsInSimulcastMode = currentCodec === CodecMimeType.AV1 || currentCodec === CodecMimeType.VP9
```

Comment:
I think I've seen a similar check elsewhere. Would it make sense to have some `SIMULCAST_MODE_CODECS` array and check fi the codec in question is in there?